windowing.py

```diff
@@ -205,3 +207,87 @@ def cropped_np(np_sample, start, end):
     end_index = np.where(nanos == end)
     cropped = [np_sample[:,  start_index[0][0]:end_index[0][0] + 1]]
     return cropped
+
+
+def get_start_time(lift_windows, rows, window_size, stride, variance_threshold):
+    """
+        Description: Returns the precise start time if it is not None or returns the start found 
+                     from find_initial_lift_times 
```

Comment:
Not exactly because inside of precise start time I return None if there it can't find a more precise start time. It was useful in the beginning to know just how often it would not find a better start time and I thought it could be interesting to leave it in incase someone wanted to know that. Should I remove it?

Reply:
my point is, that it can't find an update to the start_timestamp making it the initial start timestamp the precise timestamp. To me the current formulation is a bit confusing, since I'd first have to figure out why the precise start time could be none.

Reply:
Oh ok, so should I  make the find_precise_start_time always return the initial start time if it can't find a more precise one? Would that be a less confusing way to set it up? If it can't find a more precise start time it just means that there is not enough variance in the window to meet the given threshold.